sdk/src/Services/CloudWatchLogs/Generated/Model/_bcl45+netstandard/GetLogEventsPaginator.cs

```diff
@@ -70,7 +70,7 @@ IEnumerable<GetLogEventsResponse> IPaginator<GetLogEventsResponse>.Paginate()
                 nextToken = response.NextForwardToken;
                 yield return response;
             }
-            while (!string.IsNullOrEmpty(nextToken));
```

Comment:
The changes look good, but can you double check if the other SDKs (Java / JS) do something different for `NextBackwardToken`? (asking because this API allows customers to paginate in both directions)